storage.go

```diff
@@ -35,7 +35,7 @@ import (
 // with the cluster.
 //
 // The Load, Delete, List, and Stat methods should return
```

Comment:
What does `ErrNotExist` mean exactly for List? In the `filestorage` implementation, `ErrNotExist` is returned if the prefix is not a valid path. This seems a bit strange 🤔 .

Reply:
Yeah; if you try to list the contents of a folder that doesn't exist, it should return ErrNotExist. Is that strange?